File: Java/프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.java
class Solution {
    public String solution(String my_string, int num1, int num2) {
        char[] ch = new char[my_string.length()];
        for(int i=0; i<ch.length;i++){
            ch[i]=my_string.charAt(i);
        }
        char temp='a';
        temp=ch[num1]; // e
        ch[num1]=ch[num2];
        ch[num2]=temp;
        String answer = String.valueOf(ch);
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.js
function solution(my_string, num1, num2) {
    var answer = '';
    var arr = new Array(my_string.length);
    for(var i=0; i<my_string.length; i++){
        arr[i]= my_string.charAt(i);
    }
    var temp=arr[num1];
    arr[num1]=arr[num2];
    arr[num2]=temp;
    for(var i=0; i<arr.length; i++){
        answer+=arr[i];
    }
    return answer;
}